app/models/service_template_transformation_plan_task.rb

```diff
@@ -246,6 +246,11 @@ def vm_resource
     miq_request.vm_resources.find_by(:resource => source)
   end
 
+  def virtv2v_running?
+    return false if options[:virtv2v_started_on].blank? || options[:virtv2v_finished_on].present? || options[:virtv2v_wrapper].blank?
```

Comment:
If you invert this you can drop the return 
`options[:virtv2v_started_on].present? && options[:virtv2v_finished_on].blank? && etc...`